### PR TITLE
Move lib

### PR DESCRIPTION
**Pull request by sachs7**
- import library as per PEP8.

### CODE REVIEW

**Comment by stephendonner** [comment]
Thanks for the PR, @sachs7 - really great to have you contributing!  Let us know if we can help in any way!

**Comment by sachs7** [comment]
Thanks, @stephendonner, it is my first PR to open source community. I will look at other open issues and try to fix some.

**Comment by stephendonner** [comment]
Great; if you're interested, @sachs7 - would you mind going through http://firefox-test-engineering.readthedocs.io/en/latest/ and either filing Issues (https://github.com/mozilla/firefox-test-engineering/issues) or pull requests to fix anything obviously unclear/missing?

**Comment by sachs7** [comment]
@stephendonner going through the links which you shared above.

File: tests/test_group.py [hunk 1]
```diff
@@ -2,11 +2,12 @@
...
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import time
-from random import randrange
+import uuid
 
 import pytest
 
+from random import randrange
+
```
Comment (davehunt) [inline tests/test_group.py]: There was no need to move this line down. Both `uuid` and `random` are standard library modules, so according to PEP8 these should be grouped together. See https://www.python.org/dev/peps/pep-0008/#imports for more details.

**Comment by davehunt** [comment]
Thanks @sachs7! So you know, there's no need to create new branches and open new pull requests after addressing review comments. You can simply add new commits to your existing branch and push it. GitHub will then update the open pull request with the new commits.

**Comment by sachs7** [comment]
Thanks, @davehunt. It was a good learning and I will improve on it. Thanks, for your suggestion and will keep it in mind.

**Pull request merged**